src/lib/Navbar.svelte

```diff
@@ -1,4 +1,11 @@
 <nav class="navbar">
+	<script type="application/ld+json">
```

Comment:
This isn't proper with svelte (and may break the page!), so please remove this.

Even if it was allowed, `cssSelector` is incorrect.

Reply:
removed